src/ProjectTemplates/test/BlazorServerTemplateTest.cs

```diff
@@ -24,7 +24,7 @@ public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory, BrowserFix
 
         public Project Project { get; private set; }
 
-        [ConditionalFact]
+        [ConditionalFact(Skip = "This test ran for over an hour")]
```

Comment:
It's starting to feel like all the template tests can timeout.

Reply:
I don't think so, when you quarantined the grpc one you didn't do these 2 even though they showed up. And now they're showing up on their own. The rest of the template tests have yet to show these kinds of timeouts.